packages/manager/src/features/CloudPulse/Widget/components/Zoomer.tsx

```diff
@@ -20,28 +20,37 @@ export const ZoomIcon = React.memo((props: ZoomIconProperties) => {
   const ToggleZoomer = () => {
     if (props.zoomIn) {
       return (
-        <ZoomInMap
+        <IconButton
           sx={{
             color: theme.color.grey1,
             fontSize: 'x-large',
             height: '34px',
+            padding:0,
           }}
+          aria-label="zoom-in"
```

Comment:
```suggestion
          aria-label="Zoom In"
```
Similar suggestion to my other comment, but something more readable for our `aria-label`s would be good. Still, this is a huge improvement. Thanks!